packages/utils/address.ts

```diff
@@ -38,9 +38,7 @@ export const getSvmAddressFromLux = (luxBech32: string): PublicKey => {
     throw `${luxBech32} is not a valid lux addresss`
   }
 
-  return new PublicKey(
-    getSvmAddress(Buffer.from(bech32.fromWords(bech32.decode(luxBech32).words)))
-  )
+  return new PublicKey(getSvmAddress(Buffer.from(bech32.fromWords(bech32.decode(luxBech32).words))))
```

Comment:
We can rewrite parseFlux(s) that wraps `Buffer.from(bech32.fromWords(bech32.decode(luxBech32).words)))`